### ISSUE TITLE
Aria-labels translation

### ISSUE THREAD

**Issue by CarlEricLavoie** [SAP/spartacus#6492]
Aria-labels and other hidden attributes should be translated. As of now those attributes are hardcoded in the HTM.

see : https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/projects/storefrontlib/src/cms-components/product/product-list/container/product-list.component.html

as an example.

## Environment Details

- Spartacus:  1.5.0

## Expected Results
Aria-labels should be translated using the i18n mechanism.

**Comment by kacperknapik** [comment]
We plan to handle ARIA-labels/attributes in the future, when we implement more accessibility features. Currently we don't handle them at all.
